packages/language-server/src/completion-items.ts

```diff
@@ -27,41 +27,47 @@ import {
 } from "@ui5-language-assistant/xml-views-completion";
 import { ui5NodeToFQN } from "@ui5-language-assistant/logic-utils";
 import { getNodeDocumentation, getNodeDetail } from "./documentation";
+import { Settings } from "@ui5-language-assistant/settings";
 import { assertNever } from "assert-never";
 
-export function getCompletionItems(
-  model: UI5SemanticModel,
-  textDocumentPosition: TextDocumentPositionParams,
-  document: TextDocument
-): CompletionItem[] {
-  const documentText = document.getText();
+export function getCompletionItems(opts: {
+  model: UI5SemanticModel;
+  textDocumentPosition: TextDocumentPositionParams;
+  document: TextDocument;
+  documentSettings: Settings;
```

Comment:
is the `Settings` type name too generic?

Reply:
It's all the settings of the extension. Do you prefer `UI5LanguageAssistantSettings`? I think it's too long (and it's already imported from `@ui5-language-assistant/settings`)

Reply:
its subjective, the import would be huge but the usage more clear.
Maybe `UI5Settings`?

Reply:
or UI5EditorSettings?

Reply:
`UI5LanguageAssitantSettings` looks way too long and I feel like the others aren't accurate... it's the settings for the `UI5LanguageAssitantSettings` vscode extension and language server. Not the editor and not UI5 (`UI5Settings` sounds like the configuration of UI5 inside a project, e.g. the version, which might be part of our settings in the future).
Inside this repository I think it's ok that it's generic because it really is all the settings we have. And it seems unlikely that they will be used from outside this repo.